docs/index.md

```diff
@@ -223,13 +223,14 @@ SIMPLE_SETTINGS = {
     'DYNAMIC_SETTINGS': {
         'backend': 'consul',
         'host': 'locahost',
-        'port': 8500
+        'port': 8500,
+        'prefix': 'mynamespace/'      # Prefixes all key requests in consul with this value
```

Comment:
Can you add this comment as an observation in doc instead of a simple python commentary in code snippet ?